webapp/src/main/java/ar/edu/itba/paw/webapp/controllers/old/OldStudentController.java

```diff
@@ -0,0 +1,783 @@
+	                                       @ModelAttribute("user") UserSessionDetails loggedUser,
+	                                       @Valid @ModelAttribute("courseFilterForm")
+	                                           final CourseFilterForm courseFilterForm,
+	                                       final BindingResult errors) {
+
+		if (!loggedUser.hasAuthority("ADD_INSCRIPTION")
+				|| (loggedUser.getId() != docket && !loggedUser.hasAuthority("ADMIN"))) {
+			LOGGER.warn("User {} tried to add inscription for student {} and doesn't have ADD_INSCRIPTION authority [GET]", loggedUser.getDni(), docket);
+			return new ModelAndView(UNAUTHORIZED);
+		}
+
+		if (!model.containsAttribute("inscriptionForm")) {
+			model.addAttribute("inscriptionForm", new InscriptionForm());
+		}
+
+		final ModelAndView mav = new ModelAndView("courses");
+
+		final CourseFilter courseFilter = new CourseFilter.CourseFilterBuilder().
+				id(courseFilterForm.getId()).keyword(courseFilterForm.getName()).build();
+
+		final List<Course> courses = loadAvailableInscriptionsForDocketByFilter(docket, mav, errors, courseFilter);
+		if (courses == null) {
+			return NOT_FOUND_MAV;
+		}
+
+		final Student student = studentService.getByDocket(docket);
+		if (student == null) {
+			return NOT_FOUND_MAV;
+		}
+
+		mav.addObject("student", student);
+		mav.addObject("section2", "inscription");
+		mav.addObject("courseFilterFormAction", "/students/" + docket + "/inscription");
+		mav.addObject("inscriptionFormAction", "/students/" + docket + "/inscription");
+		mav.addObject("subsection_enroll", true);
+		mav.addObject("courses", courses);
+		mav.addObject("docket", docket);
+		return mav;
+	}
+
+//	@RequestMapping(value = "/students/{docket}/inscription", method = RequestMethod.POST)
+//	public ModelAndView studentInscription(@PathVariable final int docket,
+//										   @Valid @ModelAttribute("inscriptionForm") InscriptionForm inscriptionForm,
+//										   final BindingResult errors,
+//										   final RedirectAttributes redirectAttributes,
+//										   @ModelAttribute("user") UserSessionDetails loggedUser) {
+//
+//		if (!loggedUser.hasAuthority("ADD_INSCRIPTION")
+//				|| (loggedUser.getId() != docket && !loggedUser.hasAuthority("ADMIN"))) {
+//			LOGGER.warn("User {} tried to add inscription for student {} and doesn't have ADD_INSCRIPTION authority [POST]", loggedUser.getDni(), docket);
+//			return new ModelAndView(UNAUTHORIZED);
+//		}
+//		if (errors.hasErrors()){
+//			redirectAttributes.addFlashAttribute("org.springframework.validation.BindingResult.inscriptionForm", errors);
+//			redirectAttributes.addFlashAttribute("inscriptionForm", inscriptionForm);
+//			return new ModelAndView("redirect:/students/" + docket + "/inscription");
+//		}
+//
+//		boolean done = studentService.enroll(inscriptionForm.getStudentDocket(), inscriptionForm.getCourseId());
+//
+//		if (!done) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)